cognito-pre-token-generation-openid/src/main/java/no/unit/nva/cognito/UserSelectionUponLoginHandler.java

```diff
@@ -219,9 +221,11 @@ private List<AttributeType> customerSelectionClaims(AuthenticationInformation au
         return List.of(currentCustomerClaim, currentTopLevelOrgClaim, usernameClaim, personAffiliationClaim);
     }
     
-    private String createAllowedCustomersString(Collection<CustomerDto> allowedCustomers) {
+    private String createAllowedCustomersString(Collection<CustomerDto> allowedCustomers, String feideDomain) {
         var result = allowedCustomers
                          .stream()
+                         .filter(customer -> !Objects.nonNull(feideDomain) || customer.getFeideOrganizationDomain()
```

Comment:
For clarity's sake, extract the condition to a method — from what I understand, something like:
(note "Objects::isNull")

```
    private static boolean isNotFeideRequestOrIsFeideRequestWithCorrectDomain(String feideDomain, CustomerDto customer) {
        return isNull(feideDomain) || customer.getFeideOrganizationDomain().equals(feideDomain);
    }
```

Reply:
Good point. Fixed.